tools/update-docs-tutorial-screenshots.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+STARTING_DIR=$PWD
+export SHOT="$PWD/tools/screenshot-xodball"
+
+updaters=($(find ./docs/tutorial/ -iname update-screenshots.sh))
+for updater in "${updaters[@]}"; do
+  echo -n "$updater ..."
+  cd $(dirname $updater)
+  ./update-screenshots.sh
+  cd "$STARTING_DIR"
+  echo " done"
+done
```

Comment:
Short variant:
```
#!/bin/sh
export SHOT="$PWD/tools/screenshot-xodball"
find ./docs/tutorial -iname "update-screenshots.sh" -execdir sh -c 'pwd && ./update-screenshots.sh' \;
```
But I think this script useless because of `update-docs-screenshots.sh` already exists.